dao/test/helpers/assert.mligo

```diff
@@ -0,0 +1,14 @@
+(* Assert contract call results in failwith with given string *)
+let string_failure (res : test_exec_result) (expected : string) : unit =
+    let expected = Test.eval expected in
+    match res with
+        | Fail (Rejected (actual,_)) -> assert (Test.michelson_equal actual expected)
```

Comment:
Thanks, that was copied from the taco shop tutorial https://gitlab.com/ligolang/ligo/-/blob/dev/gitlab-pages/docs/tutorials/taco-shop/tezos-taco-shop-test.mligo#L3
will check if it can be simplified here too